nss-day-cohort-46/holiday-road-republic-mischief
# savebutton

# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

# Testing Instructions
clicking save itinerary should populate dom with itinerary options chosen

```
git fetch --all
git checkout pp-savebutton
```

# Checklist:

- [ x] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works
